### PR TITLE
Ccalc 2994

### PR DESCRIPTION
**Pull request by zatevad**

### CODE REVIEW

File: app/uk/gov/hmrc/childcarecalculatorfrontend/controllers/FreeHoursResultsController.scala [hunk 1]
```diff
@@ -16,17 +16,32 @@
...
+
   def onPageLoad: Action[AnyContent] = withSession { implicit request =>
-    // TODO: Implement properly
-    Future(Ok("This is 15 hours results page"))
-  }
+    keystore.fetch[PageObjects]().map {
+      case Some(pageObjects) =>
+        Ok(freeHoursResults(pageObjects.childAgedThreeOrFour.getOrElse(false), pageObjects.household.location))
+      case _ =>
+        Logger.warn("Household object is missing in ExpectChildcareCostsController.onPageLoad")
```
Comment (kapil-sanghvi) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/controllers/FreeHoursResultsController.scala]: pageObjects is missing, in message

File: app/uk/gov/hmrc/childcarecalculatorfrontend/utils/CCConstants.scala [hunk 2]
```diff
@@ -25,6 +26,7 @@ trait CCConstants {
   lazy val hasPartnerKey = "hasPartner"
```
Comment (kapil-sanghvi) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/utils/CCConstants.scala]: hasPartnerKey can be removed?

File: test/uk/gov/hmrc/childcarecalculatorfrontend/TemplatesValidator.scala [hunk 3]
```diff
@@ -49,6 +49,7 @@ trait TemplatesValidator extends UnitSpec {
           }
         case ElementDetails(value, Some(id), _, _, _, _, Some(tagName), Some(tagIndex)) =>
           withClue(s"Element with id '${id}', tag '${tagName}' and tagIndex '${tagIndex}': ") {
+            println(s"$id>>$tagName>>>$tagIndex>>>>$value")
```
Comment (kapil-sanghvi) [inline test/uk/gov/hmrc/childcarecalculatorfrontend/TemplatesValidator.scala]: println??

File: test/uk/gov/hmrc/childcarecalculatorfrontend/views/FreeHoursResultsSpec.scala [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        verifyPageContent(
+          List(
+            ElementDetails(tagName = Some("li"), tagIndex = Some(0), value = applicationMessages.messages(s"free.hours.results.entitled.${loc}")),
+            ElementDetails(id = Some("free-hours-results-entitled"), tagName=Some("p"), tagIndex=Some(0),
+              value = applicationMessages.messages(s"free.hours.results.entitled.info.${loc}")),
+            ElementDetails(id = Some("free-hours-results-entitled"), tagName=Some("p"), tagIndex=Some(1),
+              value = applicationMessages.messages(s"free.hours.results.entitled.info.no.costs"))
+          )
+        )
+        if(loc != LocationEnum.ENGLAND) {
```
Comment (kapil-sanghvi) [inline test/uk/gov/hmrc/childcarecalculatorfrontend/views/FreeHoursResultsSpec.scala]: why we need this if statement here?

Comment (zatevad) [inline test/uk/gov/hmrc/childcarecalculatorfrontend/views/FreeHoursResultsSpec.scala]: This is fine

**Comment by zatevad** [comment]
Fixed comments

**Pull request merged**